src/scripts/main.js

```diff
@@ -1 +1,17 @@
 'use strict';
+
+const image = document.getElementById('largeImg');
+
+const links = document.querySelectorAll('a');
+
+links.forEach((link) => {
+  const img = link.querySelector('img');
+
+  link.setAttribute('href', '#');
```

Comment:
Setting the href attribute of all links to '#' will prevent the default navigation behavior of these links. If this is not the intended behavior, consider removing this line or conditionally setting the href attribute based on specific criteria.